pkgs/development/tools/kwok/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "kwok";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "kubernetes-sigs";
+    repo = "kwok";
+    sha256 = "sha256-BTlg9zg3S1fwG6Gb4PYAcnlgPNC8sGkP1K9wYmuOPmU=";
+  };
+
+  vendorHash = "sha256-Wr7MZ2LLxKE84wmItEnJj8LhxMca4rooadiv4ubx/38=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  subPackages = [ "cmd/kwok" "cmd/kwokctl" ];
+
+  CGO_ENABLED = 0;
+
+  ldflags = [ "-s" "-w" ];
+
+  doCheck = false;
```

Comment:
Please either remove this or explain why.

Reply:
Sorry, that is just for removing debug symbols from the binary. If it's not needed I will take it out.